6/alpine/Dockerfile

```diff
@@ -2,20 +2,19 @@ FROM alpine:3.9
 
```

Comment:
Since this hits EOL on the 30th, maybe better to skip instead of pushing a new version

Reply:
We can wait after the 30th an rebase this after version 6 has been removed.

Reply:
Done